src/phone-input/styled-components.js

```diff
@@ -31,28 +35,33 @@ export const StyledFlag = styled(
   },
 );
 
-export const StyledRoot = withStyle(
+export const StyledRoot = withStyle<typeof SelectStyledRoot, SizeStyleProps>(
   SelectStyledRoot,
-  ({size = SIZE.default, $theme: {sizing}}) => {
+  (props: SizeStyleProps): StyleObject => {
```

Comment:
Why is `StyleObject` necessary here?

Reply:
I was testing something out and landed this, reverting now